src/main.ts

```diff
@@ -35,7 +36,9 @@ export function main(options: IMainOptions) {
     }
 
     app.use(GRAPHQL_ROUTE, bodyParser.json(), graphqlExpress({
-        context: {},
+        context: {
+            persons
```

Comment:
you can import findPerson in main and pass it here in context along with persons